Java/app/src/main/java/com/taboola/sdk4example/sdk_classic/MetaClassicUnitFragment.java

```diff
@@ -0,0 +1,79 @@
+package com.taboola.sdk4example.sdk_classic;
+
+import static com.taboola.sdk4example.Const.META_FEED_MODE;
+import static com.taboola.sdk4example.Const.META_FEED_PLACEMENT_NAME;
+import static com.taboola.sdk4example.Const.META_WIDGET_MODE;
+import static com.taboola.sdk4example.Const.META_WIDGET_PLACEMENT_NAME;
+
+import android.os.Bundle;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+
+import androidx.annotation.Nullable;
+
+import com.facebook.ads.NativeAdLayout;
+import com.taboola.android.MetaPlacementProperties;
+import com.taboola.android.TBLClassicPage;
+import com.taboola.android.TBLMetaClassicUnit;
+import com.taboola.android.TBLPublisherInfo;
+import com.taboola.android.Taboola;
+import com.taboola.android.annotations.TBL_PLACEMENT_TYPE;
+import com.taboola.android.listeners.TBLClassicListener;
+import com.taboola.sdk4example.Const;
+import com.taboola.sdk4example.R;
+import com.taboola.sdk4example.tabs.BaseTaboolaFragment;
+
+import java.util.HashMap;
+
+public class MetaClassicUnitFragment extends BaseTaboolaFragment {
+
+    private View mRootView;
+    private NativeAdLayout adContainerTop;
```

Comment:
@voltfmanSlava - Should this be prefixed with m, **mAdContainerTop**?

Reply:
@tomer-br-taboola updated